### ISSUE TITLE
Allow oauth passthrough

### ISSUE THREAD

**Issue by dantebarba** [rclone/rclone-webui-react#106]
Hi.

I'm currently running webui inside a docker container. I use treafik + traefik-oauth for authentication, so I usually disable password protection on any kind of backend, because they are already protected by the reverse proxy. Unfortunately there is no way of disabling authentication, or at least providing a header to authenticate. It would be great if it could be added.

**Comment by negative0** [comment]
Hi. 

rclone api uses basic auth for authentication, you can specify the username password in the base 64 encoded authorization header which could be passed by traefik. So traefik will authenticate for you.

Is it possible? I have not worked much with traefik

**Comment by dantebarba** [comment]
Yup, ended up doing it that way. But it would be cool if the option for reverse proxy auth was added.
